### ISSUE TITLE
Balancer script expects that every OSD class is present in cluster

### ISSUE THREAD

**Issue by Dragonn** [TheJJ/ceph-balancer#9]
Following happened to me when I migrated all data from `ssd` class OSDs to `nvme` ones, but I didn't remove class `ssd` from CRUSH. I didn't do that because I might re-add them later and I don't want to get rid of existing CRUSH rules.

```
./placementoptimizer.py -v balance --max-pg-moves 20 | tee -a "$(date +%Y-%m-%d)_balance-upmaps_1"
[2022-04-06 20:10:41,020] gathering cluster state via ceph api...
Traceback (most recent call last):
  File "./placementoptimizer.py", line 269, in <module>
    class_df_stats = CLUSTER_STATE["df_dump"]["stats_by_class"][crush_class]
KeyError: 'ssd'
```

I completely understand what's happenning here, I just expect the script to simply ignore classes without OSDs.

**Comment by TheJJ** [comment]
I hope I've fixed this in c692b0d58d0a271bf5ab7c56bdae1eb815c7294f, please try :)

**Comment by Dragonn** [comment]
Since I have already get rid of all `ssd` class and rules (rule cannot be created without OSDs of given class), I am unable to test it properly right now.

Nevertheless it looks fixed to me, thank you! I am going to test it properly in near future and reopen this issue if any further issues occurs.
